index.js

```diff
@@ -197,7 +199,37 @@ function init (ssb, config) {
       const data = keystore.group.get(id)
       if (!data) return cb(new Error(`unknown groupId ${id})`))
 
-      cb(null, data)
+      // find if this group has any parent links
+      scuttle.link.findGroupBySubgroupId(id, (err, links) => {
+        if (err) return cb(err)
+
+        if (!links || !links.length) return cb(null, data) // not a subgroup
+
+        const groupData = {
+          ...data,
+          parentGroupId: links[0].parentGroupId // NOTE: here we assume that there can only be one parent group
```

Comment:
lets at least console.warn if `links.length > 1` ... and the links point at different things